package/Dockerfile

```diff
@@ -1,6 +1,6 @@
-FROM registry.suse.com/bci/bci-base:15.5 AS build
+# syntax=docker/dockerfile:1.7.1
```

Comment:
Ｗhat's the purpose of the external Dockerfile frontend?

Reply:
This one: https://github.com/rancher/support-bundle-kit/pull/110#discussion_r1569937958